# Touching point pair for grasping
geometry_msgs/Point point1
geometry_msgs/Point point2